libraries/persistent_store/src/fragment.rs

```diff
@@ -24,6 +24,7 @@ use alloc::vec::Vec;
 use core::ops::Range;
 
 /// Represents a sequence of keys.
+#[allow(clippy::len_without_is_empty)]
```

Comment:
Interesting lint :-) Thanks for the fix.

Reply:
I was wondering if I should add the function itself, but `len` is O(1), so it should be okay.

Reply:
Yes, and also we would never call this function (because it's useless for keys to be empty), so it's silly. The lint is just over-triggering.